### ISSUE TITLE
Startup failed in modpack, no suspected mods. 

### ISSUE THREAD

**Issue by Lijosu** [SteamPunkNation/CreateHorsePower#4]
I apologize if this is too little to work with, i am new to modding. If anything else is needed, please elaborate.

Crash report enc

[crash-2024-11-29_01.35.08-fml.txt](https://github.com/user-attachments/files/17953461/crash-2024-11-29_01.35.08-fml.txt)

**Comment by SteamPunkNation** [comment]
Try removing the Flywheel mod, it appears in the logs that's whats causing render issues with CreateHorsePower. If that does not work send me a mod list and I will look at it in more detail and do a potential fix it Flywheel mod is incompatible or has some other issue.

**Comment by Lijosu** [comment]
> Try removing the Flywheel mod, it appears in the logs that's whats causing render issues with CreateHorsePower. If that does not work send me a mod list and I will look at it in more detail and do a potential fix it Flywheel mod is incompatible or has some other issue. Also try to use older/newer versions of Flywheel if possible to see if the different versions could have different results.
> 
> No worries if your new to modding, you will always be welcomed here at least to ask any type of questions. :)

You are right that i have Flywheel installed, but it does not appear in my modpack's content in curseforge, so it seems like a mod has come packaged with it and i am unsure which one. 
I can provide a modlist, as enclosed below, though i am unsure how much help i can be beyond that. I could go through all of these and enable/disable them until the modpack loads, i suppose? That could take a long time. 

[modlist.txt](https://github.com/user-attachments/files/17968360/modlist.txt)

**Comment by SteamPunkNation** [comment]
Try removing the Flywheel mod and then run your modpack, if another error happens just send your logs again here.

**Comment by Lijosu** [comment]
I cannot find flywheel in my modlists or my mod folder. I think "create" might by coming packaged with it, but if I disable create, your mod will not work because it needs create to function(?)

[crash-2024-12-02_05.03.48-fml.txt](https://github.com/user-attachments/files/17972783/crash-2024-12-02_05.03.48-fml.txt)

**Comment by Lijosu** [comment]
Here you go.
[RealMedi-0.3.1.zip](https://github.com/user-attachments/files/17987370/RealMedi-0.3.1.zip)

**Comment by SteamPunkNation** [comment]
Hi there, upon testing I recommend to do the following to fix your issues.

1. Update your mod-pack forge version to at least 47.3.6 as that is the minimum requirement for CreateHorsePower.
2. It appears the issue stems from the mod Smooth Boot (Reloaded) when the smoothboot.json file is deleted in the config folder it fixes the issue. This appears to be a potential mod conflict and I would recommend if possible to remove that mod or to not include its config in your mod-pack's final release.

Do let me know if you have any other issues!

**Comment by Lijosu** [comment]
I have removed Smooth Boot and tried both forge 47.3.6 and 47.3.12 to no avail.
I have enclosed the 47.3.12 version and its crash log below. 
[crash-2024-12-03_23.48.30-fml.txt](https://github.com/user-attachments/files/18000635/crash-2024-12-03_23.48.30-fml.txt)
[RealMedi-0.3.2.zip](https://github.com/user-attachments/files/18000608/RealMedi-0.3.2.zip)

**Comment by SteamPunkNation** [comment]
Hi there apologies for the wait, I have just tested your mod pack on three different systems and all seem to have no issues running the mod pack. I've tested it on a MacBook and on two different Windows PC's. I do apologize that I cannot seem to find a solution for you. Only thing I can recommend at this point is to check your Java version, update Windows, update your GPU drivers, and potentially allocate more ram to your Minecraft instance.

Best,
Steam
